CONTRIBUTING.md

```diff
@@ -126,6 +126,9 @@ information about logging and error handling.
     * Public members of structs and classes do not have the `m_` prefix
 * Namespaces in `lower_snake_case` : `my_namespace`
 * Aliases have a `_t` postfix : `using FooString_t = iox::cxx::string<100>;`
+* Objects created from a method returning a `cxx::optional<Foo>` shall be named `maybeFoo`
+* Objects created from a method returning a `cxx::expected<Foo, FooError>` shall
+contain the name `result` e.g. `fooResult`
```

Comment:
This is not quite what we do, at least not what I recall. It is more like `auto getChunkResult = blup.getChunk(...);`

Reply:
True. What do you think about?
```suggestion
* Objects created from a method returning a `cxx::expected<Foo, FooError>` shall
contain the name `result` e.g. `somehtingResult`
```

Reply:
Added an explicit example

Reply:
I would add a concrete example like
```suggestion
* Objects created from a method returning a `cxx::expected<Foo, FooError>` shall
contain the name `result` e.g. `allGloryToTheHypnotoadResult` for a call to `allGloryToTheHypnotoad()`
```